eventkitd/path.go

```diff
@@ -27,8 +27,9 @@ func pathEscapeTo(val string, out *strings.Builder) {
 	}
 }
 
-func computePath(timestamp time.Time, scope []string, name string) string {
+func computePath(base string, timestamp time.Time, scope []string, name string) string {
```

Comment:
i don't expect eventkitd to have broad cross-platform support. being able to stuff data into file names twice as efficiently is worth losing macos or windows support imo

Reply:
You can still do it, instead of adding capital letters, make them prefixed with `!` like in go module cache. e.g. `helloWorld -> hello!world`. Many of our devs work on macs (and tiny bit on Windows) -- so this would break their dev environment.